File: atributos/ListaPrivados.java
import java.lang.reflect.Field;
import java.lang.reflect.Modifier;

public class ListaPrivados {

	public static void main(String[] args) {
		ObjetoSecreto oSecreto = new ObjetoSecreto("senha super secreta");
		Field campos[] = ObjetoSecreto.class.getDeclaredFields();
		for (Field campo : campos)
			if (Modifier.isPrivate(campo.getModifiers())) { // só campos privados!
				System.out.println(campo.toString());
				campo.setAccessible(true); // arrombamos a porta
				try {
					String valor = (String) campo.get(oSecreto);
					System.out.println("\t" + campo.getName() + " -> " + valor);
				}
				catch (IllegalAccessException e) { 
					// esta exceção nao acontece porque fizemos setAcessible(true)
					System.err.println(e);
					System.exit(1);
				}	
			}
	}
}

File: atributos/java/ListaPrivados.py
from java.lang.reflect import Modifier
import ObjetoSecreto

oSecreto = ObjetoSecreto('senha super secreta')
campos = ObjetoSecreto.getDeclaredFields()
for campo in campos:
	if Modifier.isPrivate(campo.getModifiers()): # so campos privados!
		print campo
		campo.setAccessible(True) # arrombamos a porta
		print '\t', campo.getName(), '=', campo.get(oSecreto)
